set_test.go

```diff
@@ -208,6 +208,54 @@ func TestSetEqual(t *testing.T) {
 	}
 }
 
+func TestSetFirst(t *testing.T) {
+	t.Parallel()
+
+	var s Set
+	less := func(a, b interface{}) bool { return a.(int) < b.(int) }
+	assert.Panics(t, func() { s.First(less) })
```

Comment:
Would be better: assert.Panics(t, func() { s.First(less) }, "empty set")

Reply:
added the message